src/pages/ChooseFilterPage.tsx

```diff
@@ -99,7 +99,7 @@ function ChooseFilterPage({
   return (
     <Layout
       meta={{
-        title,
+        title: `${chapterTitle} - ${title} - Choose a platform`,
```

Comment:
I think we will need to check if this value exists or isn't an empty string first. For instance if we go here: https://pr-5559.d39agkzf0cwtdz.amplifyapp.com/lib/, it doesn't have a chapter title and so we have an extra dash

Reply:
Good call Tim! Added some conditional logic to check if `chapterTitle` isn't an empty string first